src/services/drive/folder/driveFolder.service.ts

```diff
@@ -50,7 +50,7 @@ class DriveFolderService {
 
     while (pendingFolders.length > 0) {
       const currentFolder = pendingFolders[0];
-      const folderContentResponse = await fileService.getFolderContent(currentFolder.folderId);
+      const folderContentResponse = await driveFileService.getFolderContent(currentFolder.folderId);
```

Comment:
here too: `drive.file.getFolderContent` ?

Reply:
Missed this ones, since they are relative, updating